easybuild/easyconfigs/f/FDTD_Solutions/FDTD_Solutions-8.16-fix-interconnect-api-path.patch

```diff
@@ -0,0 +1,16 @@
+# Fix hardcoded paths in python code
+#
+# Åke Sandgren, 2017-02-07
+diff -ru opt.orig/lumerical/fdtd/api/python/lumapi.py opt/lumerical/fdtd/api/python/lumapi.py
+--- opt.orig/lumerical/fdtd/api/python/lumapi.py	2016-09-10 02:54:25.000000000 +0200
++++ opt/lumerical/fdtd/api/python/lumapi.py	2017-02-02 12:54:37.413273068 +0100
+@@ -1,7 +1,8 @@
+ from ctypes import *
+ from numpy import *
++import os
+ 
+-INTEROPLIB = "/opt/lumerical/interconnect/api/c/libinterop-api.so.1"
++INTEROPLIB = os.environ['EBROOTINTERCONNECT'] + "/api/c/libinterop-api.so.1"
```

Comment:
Where/how is `$EBROOTINTERCONNECT` supposed to get defined?

Shouldn't this be `$EBROOTFDTD_SOLUTIONS/interconnect`?

Reply:
I originally thought it was supposed to point to Lumericals INTERCONNECT installation and its libinterop-api.so. But having looked at the corresponding file in INTERCONNECT and the fact that libinterop-api exists in fdtd_solutions too makes things a bit clearer.

Patch upcoming.